put_invoice_test.go

```diff
@@ -0,0 +1,23 @@
+package fortnox_test
+
+import (
+	"encoding/json"
+	"log"
+	"testing"
+)
+
+func TestPutInvoice(t *testing.T) {
+	req := client.NewPutInvoiceRequest()
+	req.NewPutInvoicePathParams().DocumentNumber = "1"
+	customer := req.RequestBody().Invoice
+	customer.CustomerName = "TEST"
```

Comment:
Would be better to use more real samples. 
I.e. Company name could be "Acme Inc" to test how fortnox stores multi-worded names. 
For org numbers - https://fakedetail.com/fake-company-generator/Sweden
VAT format would be then "SE" + org number + "01"